scripts/run_tests

```diff
@@ -13,6 +13,10 @@ cargo clippy --all-targets --manifest-path=./sdk/rust/Cargo.toml -- --deny=warni
 cargo test --all-targets --manifest-path=./examples/Cargo.toml
 cargo clippy --all-targets --manifest-path=./examples/Cargo.toml -- --deny=warnings
 
+# Check everything builds (except Asylo-specific targets).
+bazel build -k "${bazel_build_flags[@]}" -- //oak/...:all -//oak/server/asylo:all
```

Comment:
```suggestion
bazel_build_flags+=(
  '--keep-going'
)
bazel build "${bazel_build_flags[@]}" -- //oak/...:all -//oak/server/asylo:all
```

(use long flag name, and also it applies to any subsequent bazel command in this script)

Reply:
Done.